### ISSUE TITLE
There doesn't seem to be a way to use the latest docker images (without the "v1" tag)

### ISSUE THREAD

**Issue by evverx** [google/clusterfuzzlite#95]
As mentioned in https://github.com/google/oss-fuzz/issues/7206#issuecomment-1106677842 I'm planning to unpin CFLite but looking at the action it appears for some reason it uses tags to download the docker images:
```sh
$ git grep v1 actions/
actions/build_fuzzers/action.yml:  image: 'docker://gcr.io/oss-fuzz-base/clusterfuzzlite-build-fuzzers:v1'
actions/run_fuzzers/action.yml:  image: 'docker://gcr.io/oss-fuzz-base/clusterfuzzlite-run-fuzzers:v1'
```
Those tags are bogus in the sense that they keep rolling forward so I wonder if it's possible to remove them to make it clear that they always point to the latest images.

The idea is to always use the "main" branch and the latest images (by analogy with CIFuzz) and avoid getting bogus Dependabot updates when/if https://github.com/google/oss-fuzz/issues/7212 is implemented

**Comment by evverx** [comment]
On a somewhat related note, once I unpin CFLite the "security posture" of systemd is going to get worse even more. it would be great if OSS-Fuzz/CIFuzz/CFLite can somehow affect the scorecard fuzzing check (which is totally bogus as this point: https://github.com/ossf/scorecard/pull/1816#issuecomment-1099933020). To be honest I think that instead of adding even more bogus checks like "webhooks" scorecard should probably focus on fixing the existing checks.

**Comment by jonathanmetzman** [comment]
> As mentioned in [google/oss-fuzz#7206 (comment)](https://github.com/google/oss-fuzz/issues/7206#issuecomment-1106677842) I'm planning to unpin CFLite but looking at the action it appears for some reason it uses tags to download the docker images:
> 
> ```shell
> $ git grep v1 actions/
> actions/build_fuzzers/action.yml:  image: 'docker://gcr.io/oss-fuzz-base/clusterfuzzlite-build-fuzzers:v1'
> actions/run_fuzzers/action.yml:  image: 'docker://gcr.io/oss-fuzz-base/clusterfuzzlite-run-fuzzers:v1'

The tags aren't exactly bogus. We're doing this in case we make breaking changes to the API in v2

**Comment by jonathanmetzman** [comment]
> On a somewhat related note, once I unpin CFLite the "security posture" of systemd is going to get worse even more. it would be great if OSS-Fuzz/CIFuzz/CFLite can somehow affect the scorecard fuzzing check (which is totally bogus as this point: [ossf/scorecard#1816 (comment)](https://github.com/ossf/scorecard/pull/1816#issuecomment-1099933020)).

I'm not happy about this situation either and i've complained to scorecards but it doesn't seem like they will budge. I agree I think pinning provides little security benefit, fuzzing (for C++) provides a big security benefit, so using CFLite without pinning makes a project more secure not less and that scorecards is wrong

**Comment by evverx** [comment]
> The tags aren't exactly bogus. We're doing this in case we make breaking changes to the API in v2

Agreed. After a lengthy discussion in https://github.com/google/clusterfuzzlite/pull/96 I switched to the tags. For that to fully work https://github.com/google/clusterfuzzlite/releases/tag/v1 would have to be bumped automatically though. (@oliverchang bumped it yesterday manually)

> using CFLite without pinning makes a project more secure not less and that scorecards is wrong

I have to admit I'm not even sure what `scorecard` is trying to accomplish anymore with all those checks and a few new ones. I decided to just ignore it altogether.
